### PR TITLE
Return 404 when requested resource not found/inaccessible

### PR DESCRIPTION
**Pull request by acrellin**

### CODE REVIEW

**Comment by acrellin** [comment]
Let's not merge this until we get the necessary updates in place in SP

**Comment by acrellin** [comment]
Woops, botched a rebase and overwrote the original diff here.. Will update.

**Comment by acrellin** [comment]
@stefanv @kmshin1397 Updated, ready for review

File: app/handlers/base.py [hunk 1]
```diff
@@ -309,6 +310,8 @@ def error(self, message, data={}, status=400, extra={}):
         if not (status == 404 and self.request.method == "HEAD"):
             log(f'Error response returned by [{self.request.path}]: [{message}]')
 
+        if message.startswith("Access Error: Invalid"):
```
Comment (stefanv) [inline app/handlers/base.py:313]: Is this left over from the previous version?

Comment (acrellin) [reply app/handlers/base.py:313]: Ah yes, thanks,willupdate

Comment (acrellin) [reply app/handlers/base.py]: Updated

**Review by kmshin1397 — APPROVED** [review]

**Comment by acrellin** [comment]
@stefanv how does this look to you now?

**Pull request merged**